README.md

```diff
@@ -0,0 +1,10 @@
+ItemShop
+========
+
+Sup playas.
+
+Useful Links
+------------
+- [Ashley Guide](https://github.com/libgdx/ashley/wiki/How-to-use-Ashley)
+- [LibGDX Input](https://github.com/libgdx/libgdx/wiki/Input-handling)
+- [Build Problem](http://stackoverflow.com/questions/3138384/warning-build-path-specifies-execution-environment-j2se-1-4)
```

Comment:
That is OK.  Lambda came in java 1.8 and is OK at the moment but if we want to deploy on android we may need to replace them with anonymous classes in the future. Unless of course they have fixed this since I last tried.